src/frontend.md

```diff
@@ -37,14 +35,16 @@ src/
     [exposed MF modules]
     [no nested directories]
   pages/
-    bankAccount/
-      BankAccountList/
+    bankAccount/ (entity, module, .etc)
+      BankAccountTable/
+        [no nested directories]
+        [components can be placed in the page/:folder if it is used only by components in this folder]
```

Comment:
i guess we should avoid it, here is described why: https://github.com/OsomePteLtd/principles/pull/248#discussion_r1181506411

Reply:
agree

Reply:
I believe it's a wrong guess I will show it why